notebooks/azureml_model_headers_demo/german_credit_azure_ml_certifai_pro_demo.ipynb

```diff
@@ -0,0 +1,1130 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the scan object from scratch using the ScanBuilder class with tasks and outcomes\n",
+    "\n",
+    "# First define the possible prediction outcomes\n",
+    "task = CertifaiPredictionTask(CertifaiTaskOutcomes.classification(\n",
+    "    [\n",
+    "        CertifaiOutcomeValue(1, name='Loan granted', favorable=True),\n",
+    "        CertifaiOutcomeValue(2, name='Loan denied')\n",
+    "    ]),\n",
+    "    prediction_description='Determine whether a loan should be granted')\n",
+    "\n",
+    "#  create a certifai scan object and add the certifai task created above\n",
+    "scan = CertifaiScanBuilder.create('model_auth_demo',\n",
+    "                                  prediction_task=task)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install azure-storage-blob"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os, uuid\n",
+    "from azure.storage.blob import BlobServiceClient, BlobClient, ContainerClient\n",
+    "# set credentials for azure storage account\n",
+    "az_credentials = 'REDACTED'\n",
+    "# set connection string as env var so the Certifai Library can pick it up\n",
+    "os.environ['AZURE_CONNECTION_STRING'] = az_credentials\n",
+    "# upload our evaluation dataset to an Azure Blob Storage Account Container.\n",
+    "client = BlobServiceClient.from_connection_string(az_credentials)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# upload our eval dataset to the blob storage container\n",
+    "az_container_name = 'set_container_name' # this container should already exist. You can create one from the Azure Portal\n",
+    "german_credit_eval_data_file = \"data/german_credit_eval.csv\"\n",
+    "az_german_credit_blob_name = 'az-certifai-examples/german_credit_eval.csv'\n",
+    "\n",
+    "blob_client = client.get_blob_client(container=az_container_name, blob=az_german_credit_blob_name)\n",
+    "with open(german_credit_eval_data_file, 'rb') as f:\n",
+    "    blob_client.upload_blob(f)\n",
+    "\n",
+    "container_client = client.get_container_client(az_container_name)\n",
+    "blob_list = container_client.list_blobs()\n",
+    "for blob in blob_list:\n",
+    "    print(\"\\n\" + blob.name)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### add logistic and svm models (created above) to scan object\n",
+    "\n",
+    "Additional parameters that maybe provided to the `CertifaiModel` class can be gleaned from the [API Reference for CertifaiModel](https://cognitivescale.github.io/cortex-certifai/certifai-api-ref-1.2.14/certifai.scanner.builder.html#certifai.scanner.builder.CertifaiModel)\n",
+    "\n",
+    "or `?CertifaiModel`"
```

Comment:
not sure what this is doing

Reply:
It's the standard notebook doc shortcut for python objects to understand additional parameters

Reply:
an alternative to jumping to the docs site